feature/split/src/main/java/com/blitzsplit/split/presentation/component/SplitBillValueTextField.kt

```diff
@@ -0,0 +1,29 @@
+package com.blitzsplit.split.presentation.component
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.KeyboardArrowRight
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import com.blitzplit.component.textfield.BlitzSplitTextField
+
+@Composable
+fun SplitBillValueTextField(
+    modifier: Modifier = Modifier,
+    value: String,
+    label: String,
+    onValueChange: (String) -> Unit
+) {
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Text("R$", fontWeight = FontWeight.Bold)
+        Icon(imageVector = Icons.AutoMirrored.Filled.KeyboardArrowRight, contentDescription = null)
```

Comment:
1) Bora parametrizar na equipe de quebrar linha para parâmetros nomeados, como recomendado [aqui](https://github.com/Quartel-Enterprise/bora-rachar-android-app/wiki/Line-Breaks#1-function-calls).
2) Seta algum content description para esse ícone, de acordo com o item 1.1 da competição da google (content description ajuda usabilidade para pessoas com deficiência):
> Is the solution easy and enjoyable to use for everyone, including people with disabilities? (maximum 5 points)

```suggestion
        Icon(
            imageVector = Icons.AutoMirrored.Filled.KeyboardArrowRight,
            contentDescription = stringResource(R.string.arrow_split_bill_text_field_arrow_content_description)
        )
```